.github/workflows/policy-check.yml

```diff
@@ -0,0 +1,27 @@
+name: Policy check
```

Comment:
Since this is currently more of a profanity check than policy check, and "policy check" could also refer to other things, do we want to consider a more descriptive name? E.g. "Profanity check" or "Source code policy check"

Reply:
I would like to add more source code policies into this single action.  It will get expensive to have each policy be a separate action.  Changed to "Source code policy check"